### PR TITLE
Fix mozillians failing test: test_group_invitations and test_group_description_edit

### PR DESCRIPTION
**Pull request by viorelaioia**

### CODE REVIEW

**Comment by viorelaioia** [comment]
Adhoc run: https://fx-test-jenkins-dev.stage.mozaws.net:8443/job/mozillians.adhoc/34/console

**Comment by johngian** [comment]
Please squash to a single commit.

File: pages/search.py [hunk 1]
```diff
@@ -44,11 +44,6 @@ def no_results_message_body(self):
     def advanced_options_shown(self):
         return self.is_element_visible(*self._advanced_options_locator)
 
-    def search_for(self, search_term):
```
Comment (johngian) [inline pages/search.py]: Why are you removing this? I don't see any further references in this PR.

Comment (viorelaioia) [inline pages/search.py]: @johngian , I removed this method because I noticed it is not used.

**Comment by johngian** [comment]
Please rebase and let's merge it.

**Pull request merged**